src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.impl.CsvReaderServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceTest {
+    private static final String RESOURCES_PASS = "src/main/resources/test/";
```

Comment:
you currently have resources for tests in your code. You need to create a resource folder in the tests folder